packages/react-search-ui-views/src/PagingInfo.js

```diff
@@ -10,7 +10,12 @@ function PagingInfo({ className, end, searchTerm, start, totalResults }) {
       <strong>
         {start} - {end}
       </strong>{" "}
-      out of <strong>{totalResults}</strong> for: <em>{searchTerm}</em>
+      out of <strong>{totalResults}</strong>{" "}
+      {searchTerm && (
+        <React.Fragment>
```

Comment:
You can use the [fragment shorthand](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html) `<>...</>` here instead of `<React.Fragment>...</React.Fragment>` - it's what we've been using elsewhere in the repo :)

Reply:
I just noticed after you'd mentioned. Made the change for it :)